src/Bluejay.asm

```diff
@@ -761,11 +761,10 @@ motor_start:
     mov  Flags1, A
     mov  Demag_Detected_Metric, A       ; Clear demag metric
     mov  Demag_Detected_Metric_Max, A   ; Clear demag metric max
+    mov  Ext_Telemetry_H, A             ; Clear extended telemetry data
 
     call wait1ms
```

Comment:
Why is the wait necessary here? Maybe we could add a comment.

Reply:
TBH, It is inherited code. I supose that it is there so t1_int routine can get the new values, but I don't see the reason. I will rework it

Reply:
Done